deploy/utils.js

```diff
@@ -0,0 +1,120 @@
+const https = require('https')
+const fs = require('fs')
+
+const log = {
+  info(...args) {
+    console.log('[INFO]', ...args)
+  },
+  warn(...args) {
+    console.log('[WARN]', ...args)
+  },
+  debug() {},
+  error(...args) {
+    console.error('[ERROR]', ...args)
+  }
+}
+
+function setLogLevel(logLevel = 'info') {
+  if (logLevel.toLowerCase() === 'debug') {
+    log.debug = (...args) => {
+      console.log('[DEBUG]', ...args)
+    }
+  }
+}
+
+function expandPath(path) {
+  return ['.', '/'].includes(path[0]) ? path : `${__dirname}/${path}`
+}
+
+function parseArgs(args) {
+  const parsedArgs = {}
+  let lastArgName = ''
+
+  for (const arg of args) {
+    if (arg.startsWith('--')) {
+      lastArgName = arg.slice(2)
+    } else {
```

Comment:
Yep it's ok!
it's a super simple args parser so it doesn't catch edgecases but if you supply  something like:

```--parcels genesis.json --account 0x```

you'll get

```[
'--parcels',
'genesis.json',
'--account',
'0x'
]```

so that else it's assigning the last `--name` name to the value